src/commands/kv/delete_namespace.rs

```diff
@@ -0,0 +1,25 @@
+use cloudflare::apiclient::APIClient;
+
+use cloudflare::workerskv::remove_namespace::RemoveNamespace;
```

Comment:
should this be delete_namespace and DeleteNamespace for consistency?

Reply:
these come from cloudflare-rs, which i believe based its naming off of the api docs. i don't think it's necessary to make that upstream change...